stip/stip-001-burn-election.md

```diff
@@ -0,0 +1,127 @@
+# Abstract
+
+This proposal describes a mechanism for single-leader election using
+burn-mining. Leader election is intimately connected to the concept of
+chain selection-- as leaders must be able to choose their chain tip, and
+that choice is impacted by network participants' rules for chain selection.
+
+# Motivation
+
+Leader election on the Stacks chain enables:
+
+1. Near instantaneous confirmation of Stacks transactions
+2. High validation throughput
+3. An open leadership set
+4. Opportunity to participate without mining hardware
+5. Ability to migrate to a separate burn chain in the future
+
+# Election
+
+The basic structure for leader election through burn-mining is that
+for some Stacks block _N_, the leader is selected via some function of
+that leader's total cryptocurrency burnt in a block _N'_ on the
+underlying burn chain (or over a window of blocks).
```

Comment:
To clarify, I think the plan was always to use a window of blocks.  Maybe the wording should read something like:

"""
The basic structure for leader election through burn-mining is that
￼ for some Stacks block _N_, the leader is selected via some function of
￼ that leader's total cryptocurrency burnt in a block window _N'_start_ to _N'_end_ on the
￼ underlying burn chain, where _N'_start_ <= _N'_end_ < N.
"""

Reply:
I think it's much easier to describe in this document as a single block -- and have the windowing described basically as an extension. It's the same as saying, the "burn value" for address X at Block N is equal to the sum of their burns over the prior W blocks.

Reply:
Sounds good :+1: